WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2167,11 +2180,32 @@ static GSourceFuncs _handlerIntervention =
                 }
                 g_mkdir_with_parents(wpeDiskCachePath, 0700);
 
+#if WEBKIT_CHECK_VERSION(2, 38, 0)
+                if (_config.NetworkProcessLimit.IsSet() == true) {
```

Comment:
What if config doesnt have the limits set? Shouldnt the default values be set?

Reply:
As discussed, WebKitBrowser has built-in default values. Those apply when config is not set.